build/jobs/e2e-tests-extract.yml

```diff
@@ -0,0 +1,10 @@
+parameters:
+- name: version
+  type: string
+
+steps:
+  - task: ExtractFiles@1
+    displayName: 'Extract E2E Test Binaries'
+    inputs:
+      archiveFilePatterns: '$(System.ArtifactsDirectory)/IntegrationTests/Microsoft.Health.Fhir.${{ parameters.version }}.Tests.E2E.zip'
+      destinationFolder: '$(Agent.TempDirectory)/E2ETests/'
```

Comment:
Should this folder also have the parameters' version as part of the folder name? 
Can it cause conflicts?

Reply:
From all the paths I can see, we run only one test package on an agent, so doesn't seem like its strictly needed